src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE `home_work1` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
```

Comment:
what mean this comment?

Reply:
it generated in MySql Workbench. So i just puted this to the file without any changes